src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -810,6 +810,21 @@ private InputsAndSuccessFlag selectInputs(TypedOperation operation) {
       Variable randomVariable = varAndSeq.var;
       Sequence chosenSeq = varAndSeq.seq;
 
+      // Fuzz the inputs for method calls and constructors to increase tests diversity.
+      // See randoop.generation.GrtFuzzing for details.
+      int chosenSeqSize = chosenSeq.size();
+      int chosenSeqSizeAfterFuzzing = chosenSeqSize;
```

Comment:
I hope that the fuzzing can be made to be always exactly one statement, by not using StringBuffers.